source/query/queries.txt

```diff
@@ -6,6 +6,14 @@ View Past Queries
 
 .. default-domain:: mongodb
 
+Starting in |compass| 1.31, you can view your favorite and saved 
+aggregation queries on the "My Queries" view once connected to 
```

Comment:
[minor] Suggest using a guilabel for "My Queries"

```suggestion
aggregation queries on the :guilabel:`My Queries` view once connected to 
```

Reply:
Updated.